src/pages/Auth/ForgotPassword/Index.tsx

```diff
@@ -2,11 +2,11 @@ import { Button, Typography, Modal } from "antd";
 import { FieldValues, useForm } from "react-hook-form";
 import { yupResolver } from "@hookform/resolvers/yup";
 import { useNavigate } from "react-router-dom";
-import Field from "components/DefaultField/Index";
+import Field from "components/DefaultField/DefaultField";
 import { SIGN_IN_ROUTE } from "utils/consts/routeConsts";
 import { useTranslation } from "react-i18next";
 import { PRIMARY } from "utils/consts/colorConsts";
-import api from "services/auth/forgotPassword";
+import { useForgotPasswordMutation } from "services/auth/forgotPassword";
 import S from "./style";
 import ForgotPasswordSchema from "./schema";
```

Comment:
imports

Reply:
Tried to correct the whole project as well 👍🏻

Reply:
Great job! Sorry man, I understand that some of my comments refer to errors that were already in the project, I just think that such inaccuracies should be corrected